src/components/Person/PersonRow.tsx

```diff
@@ -19,6 +19,7 @@ export const PersonRow: React.FC<Props> = ({ person }) => {
     father,
   } = person;
 
+  const isFemale = sex === 'f';
```

Comment:
It's good that you moved it to a separate variable, but the "magic value" is still used - what is `f`?
It would be good to create an enum in which to describe what `female: 'f', male: 'm'`

Reply:
> It's good that you moved it to a separate variable, but the "magic value" is still used - what is `f`? It would be good to create an enum in which to describe what `female: 'f', male: 'm'`

i`m sorry, i misunderstood correctly, what`s magic value , and thought it`s bout expression in condition... :c